graphql/tests/projectSettings/results.json

```diff
@@ -119,8 +119,8 @@
         "data": {
           "projectSettings": {
             "vars": {
-              "vars": {"hello": "", "foo":  "bar", "admintest": "woo", "both": ""},
-              "privateVars": ["hello", "both"],
+              "vars": {"admintest": "woo", "both": "", "hello": "", "foo":  "bar"},
+              "privateVars": ["both", "hello"],
```

Comment:
This isn't guaranteed to ever be ordered because of how graphql works, so just flipping them isn't going to help. If we add a sort to the privateVars and adminOnly resolvers, then the tests will be guaranteed to return in a consistent order.

Reply:
returns sorted

Reply:
Ths stilaThis still has two returned private_vars, which means that order is still going to matter. Should both not be part of these anymore?

Reply:
Sorry I left it in there